__init__.py

```diff
@@ -784,7 +787,16 @@ def converse(self, utterances, lang="en-us"):
         """While an alarm is expired, check all utterances for Stop vocab."""
         if has_expired_alarm(self.settings["alarm"]):
             if utterances and self.voc_match(utterances[0], "StopBeeping"):
-                self._stop_expired_alarm()
+
+                if utterances[0].find("snooze") > -1: 
```

Comment:
Here we should use `voc_match` instead of `find` as this also handles the translation of the term.

To do this we need to put "snooze" into it's own `vocab/en-us/snooze.voc` file rather than in the `StopBeeping.voc`

This should also keep the two paths separate which seems a little cleaner, eg
```python
if self.voc_match(utterance, "StopBeeping"):
    self._stop_expired_alarm()
elif self.voc_match(utterance, "Snooze"):
    # new code
```